### DISCUSSION TITLE
Home Assistant OS 10.2

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2560] [Announcements]
## Home Assistant Operating System

* Set bind propagation for supervisor data (#2557) @mdegat01
* Bump buildroot to update linux-firmware to 20230515 (#2547) (#2553) @agners
* Bump buildroot to 2023.02.1 (#2541) @agners
* Bump buildroot to update Docker to 23.0.6 (#2539) @agners
* Add rpcbind to support NFSv3 (#2529) @agners
* Bump buildroot to update Docker to 23.0.5 (#2518) @agners
* Bump buildroot to update NetworkManager to 1.40.18 (#2516) @agners

## Open Virtual Appliance

* Enable PCIe WiFi cards to OVA (#2558) (#2559) @agners
* Linux: Update kernel 6.1.29 (#2551) @agners

## Generic x86-64

* Linux: Update kernel 6.1.29 (#2551) @agners
* Enable BCM943228HMB and other PCI based Broadcom devices (#2525) (#2530) @agners

## Hardkernel ODROID

* Linux: Update kernel 6.1.29 (#2551) @agners

## ASUS Tinker

* Linux: Update kernel 6.1.29 (#2551) @agners

## Khadas VIM Series

* Linux: Update kernel 6.1.29 (#2551) @agners

## Generic aarch64

* Linux: Update kernel 6.1.29 (#2551) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/10.2'>Home Assistant OS 10.2</a>.</em>

**Comment by tjorim** [comment]
All working well on my HA Yellow.

**Reply by andyshinn** [reply]
Are you running NVMe drive by any chance? Wondering if it is now safe to upgrade on Yellow boards running NVMe. Seems 10.0 and 10.1 there was a lingering issue still that was pinned? Is that fixed?

**Reply by tjorim** [reply]
Yes, never had an issue with my Samsung 970 EVO Plus 500GB nvme SSD.

**Comment by bcutter** [comment]
Same old story ;-)

Now released (stable), still showing beta...

![image](https://github.com/home-assistant/operating-system/assets/13799156/40bec84c-d185-4795-a7d2-571db4b726ed)

![image](https://github.com/home-assistant/operating-system/assets/13799156/837649a2-8b3f-42a6-8add-52d2a38dba87)

**Reply by agners** [reply]
As long as you are around, no need for an automation :wink: 

Thanks for the reminder!

**Comment by udivankin** [comment]
BLE is still an issue? I was forced to downgrade to 9.5 because all Bluetooth devices were becoming unavailable after a day and never reappeared until reboot+reconfiguration

**Comment by jldg** [comment]
I haven't been successful updating from 10.1 to 10.2 on my odroid m1. It either won't reboot after it tries to update or reboots but is still on 10.1

**Comment by TheHolyRoger** [comment]
Nice to see the team just ploughing on with minor updates without mention of any Pi fixes despite all these users stuck on 9.5 :)
https://github.com/home-assistant/operating-system/issues/2536
https://github.com/home-assistant/operating-system/issues/2482
https://github.com/home-assistant/operating-system/issues/2480
https://github.com/home-assistant/operating-system/issues/2485
https://github.com/home-assistant/operating-system/issues/2476

**Reply by agners** [reply]
Many of those issues are most likely issues in the firmware or kernel provided by the Raspberry Pi team. They didn't make a new Linux release for quite a while, so no updates on our side, unfortunately 😢

**Reply by bcutter** [reply]
https://github.com/home-assistant/operating-system/issues/2485 (mainly, next to https://github.com/home-assistant/operating-system/issues/2476) actually make me still feel like "Nah, let's continue staying on 9.5..." -.-
